src/Adapters/AzureManagementAdapter.ts

```diff
@@ -567,4 +571,94 @@ export default class AzureManagementAdapter implements IAzureManagementAdapter {
             return new AzureResourcesData([newRoleAssignment]);
         }, 'azureManagement');
     }
+
+    async getTimeSeriesConnectionInformation(adtInstanceIdentifier: {
```

Comment:
can we add a comment about the priority of the args? It seems like they are mutually exclusive, if both are provided, which is used?

Reply:
added strict type, yeah they are mutually exclusive, only one of them should be provided